packages/server/src/profile/profile.controller.ts

```diff
@@ -437,6 +501,23 @@ export class ProfileController {
     );
   }
 
+  @Delete(':id/delete_student')
+  @UseGuards(JwtAuthGuard)
+  async deleteStudent(
+    @Param('id') id: number,
+  ): Promise<DeletedStudentResponse> {
+    const user = await UserModel.findOne(id);
```

Comment:
I think we should add a check to verify that person with right permission can do this operation.